src/utils/TaskStatusSymbol.tsx

```diff
@@ -0,0 +1,33 @@
+import { faBan, faCheck, faExclamationCircle, faWindowClose } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Spinner from 'react-bootstrap/Spinner';
+import { Backend } from '@kopia/backend';
+import { formatDuration, cancelTask } from './functions';
+
+
+export function TaskStatusSymbol(task: Backend.Task) {
+    const st = task.status;
+    const dur = formatDuration(task.startTime, task.endTime);
+    const durMultiUnit = formatDuration(task.startTime, task.endTime, true);
+
+    switch (st) {
+        case "RUNNING":
```

Comment:
I recommend defining the strings, `"RUNNING"`, `"SUCCESS"`, `"FAILED"`, and `"CANCELED"`, as members of a TypeScript `Enum`.

For example:

Definition:

```ts
enum TaskStatus {
  RUNNING = "RUNNING",
  SUCCESS = "SUCCESS",
  FAILED = "FAILED",
  CANCELED = "CANCELED",
}
```

Usage:

```diff
- case "RUNNING":
+ case TaskStatus.RUNNING: 
```

Reference: https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Reply:
I just realized it is defined as an `Enum` already (in the file, `src/backend/ApiTypes.ts`, which is linked below), but the `Enum` is not being used in this particular situation.

https://github.com/kopia/htmlui/pull/75/files#diff-4e5aef6fa3b1521bb5aec845c4969b4a723dde435bb9f8830ebee5868a4f45b8R68